about.html

```diff
@@ -0,0 +1,138 @@
+                        planning to engage in the expanding Ethiopian market. The Expo serves as a catalyst 
+                        for collaboration and coopetition between foreign and local technology companies   </p>
+                </div>
+                <div>
+                    <p> Please contact us per Email for any further question about  ICT Expo Ethiopia 2023 !</p>
+                    <a href="">damenam@gmail.com</a>
+                </div>
+    </section>
+  
+    <section >
+        <h2> ICT Expo Ethiopia 2023  </h2>
+        <p>Innovate Ethiopia   May 14 – May 18, 2020 </p>
+            <div class="expo-eth">
+                <img class="logo-2015" src="./image/ICT-Logo-2015.jpg" alt="logoExpo">
+            </div>
+    </section>
+   
+    <section id="newsid">
+        <h2 class="see-past">see the past ICT Expo Ethiopia </h2>
+        <div class="line"></div>
+        <p class=""> ICT Expo Ethiopia is where established technology service vendors integrators, and users of ICT technology for business or public </p>
+                <div class="last-exp">
+                    <div class="first-exp">
+                        <div>
+                            <img src="./image/Past-expo1.jpg" alt="" srcset="">
+                        </div>
+                        <div class="center">
+                            <h2>2022</h2>
+                            <p>ICT Expo Ethiopia 2023 Addis Abeba</p>
+                        </div>
+                    </div>
+                    <div class="first-exp">
+                        <div>
+                            <img src="./image/Past-Expo2.jpg" alt="" srcset="">
+                        </div>
+                        <div  class="center">
+                            <h2>2021</h2>
+                            <p>ICT Expo Ethiopia 2023 Mekele</p>
+                        </div>
+                       
+                    </div>
+                </div>
+    </section>
+    <section class="reserved-section">
+        <div class="right-reserved">
+            <img class="ict-logo" src="./image/ICT-Logo-2015.jpg" alt="">
+            <p>2023 Creative Commons ICT EXPO  some Rights Reserved</p>
+        </div>
+    </section>
+    <section id="mainmenu" class="main-menu">
+        <div class="hellow-section">
+            <div>
+                <p class="hello">"Hello! Sharing word"</p>
+                <h1> ICT Expo  Innovate Ethiopia 2023 </h1>
+            </div>
+            <div>
+                <p class="body-1"> Established technology service vendors, private and 
+                    public sector decision-makers, integrators, and users of ICT technology for business </p>
+            </div>
+            <div>
+                <h3 class="date">2023.09.12(sat)</h3>
+                <p>  @May 14 – May 18, 2020 | Addis Ababa, Ethiopia</p>
+            </div>
+        </div>
+</section>
```

Comment:
- [ ] As this is the footer section, please use the semantic tag <footer> instead of the section tag. This makes your site SEO friendly 😉 
- [ ] Also notice how the footer of the about page has dark background on the desktop screens? Look at the screenshot 👇 
![image](https://user-images.githubusercontent.com/90222110/212419252-80e5b7e8-c5f3-4c2a-b65c-1d44d81893be.png)
So please change the background of footer for your site too 😃 👍